vyxal/transpile.py

```diff
@@ -4,6 +4,8 @@
 import secrets
 from typing import Union
 
+from sympy import arity
```

Comment:
Where is this used?

Reply:
Some autoimport VScode did, I had no idea about that.

```suggestion
```